eth-contracts/contracts/service/ServiceTypeManager.sol

```diff
@@ -8,27 +8,76 @@ import "../InitializableV2.sol";
 /** NOTE - will call RegistryContract.constructor, which calls Ownable constructor */
 contract ServiceTypeManager is InitializableV2, RegistryContract {
     RegistryInterface registry;
-    address versionerAddress;
+    address private controllerAddress;
+    bytes32 private governanceKey;
 
     /**
      * @dev - mapping of serviceType - serviceTypeVersion
      * Example - "discovery-provider" - ["0.0.1", "0.0.2", ..., "currentVersion"]
      */
     mapping(bytes32 => bytes32[]) public serviceTypeVersions;
 
+    // @dev List of valid service types
+    bytes32[] private validServiceTypes;
+
+    // @dev Struct representing service type stake requirements
+    struct ServiceInstanceStakeRequirements {
+        uint minStake;
+        uint maxStake;
+    }
+
+    // @dev mapping of service type to registered requirements
+    mapping(bytes32 => ServiceInstanceStakeRequirements) serviceTypeStakeRequirements;
```

Comment:
nit but can we standardize naming between l23, l24, l30 for serviceTypeStakeReqs vs serviceInstanceStakeReqs

Reply:
yes